optimum/onnxruntime/base.py

```diff
@@ -311,7 +311,7 @@ def forward(
             input_ids, past_key_values, use_torch=use_torch
         )
 
-        if self.device.type == "cuda" and self.parent_model.use_io_binding:
+        if (self.device.type == "cuda" or self.device.type == "cpu") and self.parent_model.use_io_binding:
```

Comment:
For CPU, it does not need synchronize_inputs() synchronize_outputs in line 343 below.

Reply:
Exclude synchronize_inputs() for CPU